### PR TITLE
#94: Fix a bunch of TS errors

### PR DESCRIPTION
**Pull request by mfferreira**
When `yarn compile`:
- Before: 80 errors
- Now: 0 errors

### CODE REVIEW

File: app/screens/map-screen/map-screen.tsx [hunk 1]
```diff
@@ -91,8 +91,8 @@ export const MapScreen: ScreenType = ({ navigation }: Props) => {
     useCallback(() => {
       requestLocationPermission()
 
-      if (!grantedPermission) {
-        return
+      if (grantedPermission) {
```
Comment (jrbuhl93) [inline app/screens/map-screen/map-screen.tsx:94]: I think `!grantedPermission` is correct.

Comment (mfferreira) [reply app/screens/map-screen/map-screen.tsx:94]: Not sure how that happened 🤔  Thanks for spotting!

**Comment by jrbuhl93** [comment]
Could you also add `yarn compile` to `check-code` in the Makefile

File: app/screens/earns-map-screen/earns-map-screen.story.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { action } from "@storybook/addon-actions"
```
Comment (jrbuhl93) [inline app/screens/earns-map-screen/earns-map-screen.story.tsx:1]: Can be removed

**Comment by mfferreira** [comment]
> Could you also add `yarn compile` to `check-code` in the Makefile. Maybe rename `yarn compile` to `yarn build` for back end [consistency](https://github.com/GaloyMoney/galoy/blob/26d9e44c25f1d5f4629dca71c4439f5738f9dbdf/Makefile#L55).

Updated 👌

**Pull request merged**